apps/api/src/app/events/e2e/trigger-event-topic.e2e.ts

```diff
@@ -0,0 +1,261 @@
+import {
+  IntegrationRepository,
+  JobRepository,
+  JobEntity,
+  JobStatusEnum,
+  LogRepository,
+  MessageRepository,
+  NotificationRepository,
+  NotificationTemplateEntity,
+  SubscriberEntity,
+  SubscriberRepository,
+} from '@novu/dal';
+import { ISubscribersDefine, ITopic } from '@novu/node';
+import {
+  ChannelTypeEnum,
+  StepTypeEnum,
+  IEmailBlock,
+  TemplateVariableTypeEnum,
+  TopicDto,
+  TopicKey,
+  TopicName,
+  TriggerRecipientsTypeEnum,
+  ExternalSubscriberId,
+} from '@novu/shared';
+import { SubscribersService, UserSession } from '@novu/testing';
+import axios from 'axios';
+import { expect } from 'chai';
+
+import { TriggerEventRequestDto } from '../dtos';
+
+const axiosInstance = axios.create();
+
+const TOPIC_PATH = '/v1/topics';
+const TRIGGER_ENDPOINT = '/v1/events/trigger';
+
+const createTopic = async (session: UserSession, key: TopicKey, name: TopicName): Promise<TopicDto> => {
+  const response = await axiosInstance.post(
+    `${session.serverUrl}${TOPIC_PATH}`,
+    {
+      key,
+      name,
+    },
+    {
+      headers: {
+        authorization: `ApiKey ${session.apiKey}`,
+      },
+    }
+  );
```

Comment:
Usually I prefer to move function to bottom of the file so it will be easier to scan the file on the high level first for describe block and etc... Can you move them down and maybe change the to function declarations so hoisting can solve the error before defined for you?